docs/content/en/docs/concepts/tracing-policy/selectors.md

```diff
@@ -327,6 +328,58 @@ while the whole `kprobe` call is the following:
       - "3"
 ```
 
+## Parent binaries filter
+
+Parent binaries filter is filter on current process parent binary path, which works
+in very similar way as `matchBinaries` filter does, and can be specified
+under `matchParentBinaries` field. For instance, the following `matchParentBinaries` selector
+will match only if binary `cat` was executed only from interactive shell like
+`zsh`, `bash`, `sh`:
```

Comment:
```suggestion
Parent binaries filter provides filtering based on current process parent
binary path, which works similarly to the `matchBinaries` filter. It can be
specified with the `matchParentBinaries` field. For instance, the following
`matchParentBinaries` selector will match only if binary `cat` was executed
from interactive shell like `zsh`, `bash`, `sh`:
```